src/app/public/modules/flyout/flyout-adapter.service.ts

```diff
@@ -48,4 +56,23 @@ export class SkyFlyoutAdapterService {
       iframes[i].style.pointerEvents = enable ? '' : 'none';
     }
   }
+
+  public shouldClickCloseFlyout(host: ComponentRef<SkyFlyoutComponent>, target: Node): boolean {
```

Comment:
I wouldn't be so particular if this was a private method, but can we think of a better name for this method, since internally it has no idea about a "click" event. It's checking if a click _target_ is contained within another host element. Something like `hostContainsEventTarget(host, event): boolean` (see comment below about passing in the event directly).

Reply:
Fixed